build.gradle.kts

```diff
@@ -1,8 +1,22 @@
 plugins {
     java
+    application
     kotlin("jvm") version "1.9.23"
 }
 
+application {
+    // NOTE: In case you choose a different main class, make sure to update this path
+    mainClass = "george.backend.hiring.KotlinApplicationKt"
+}
+
+tasks.jar {
+    manifest {
+        attributes["Main-Class"] = application.mainClass
+    }
+
+    from(configurations.runtimeClasspath.get().filter { it.name.endsWith("jar") }.map { zipTree(it) })
```

Comment:
🤷 This line was needed for Kotlin `.jar`s to work. Without it, Kotlin standard library was not included and running `java -jar wordcount.jar` failed with error:
```java
Exception in thread "main" java.lang.NoClassDefFoundError: kotlin/jvm/internal/Intrinsics
        at george.backend.hiring.KotlinApplicationKt.main(KotlinApplication.kt)
Caused by: java.lang.ClassNotFoundException: kotlin.jvm.internal.Intrinsics
        at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
        at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526)
        ... 1 more
```